### ISSUE TITLE
Incorrect service info page accessibility issues

### ISSUE THREAD

**Issue by esther-k** [department-of-veterans-affairs/va.gov-team#16249]
## What Happened?
1. While using a screen reader, the VA.gov text on Android is being read as "Vaw [dot] gov"
2. While using a screen reader, the programmatic page title on Android is being read as "Incorrect service info screen"
3. With VoiceOver andTalkBack enabled on iOS and Android, the back button a11yHint is not being read
     - NOTE: Should be updated with [a11y update ticket](https://app.zenhub.com/workspaces/va-mobile-app-5f205eac3155a2000f422ecb/issues/department-of-veterans-affairs/va.gov-team/17185)

## Specs:
- Android LG LM-Q720 Stylo 5 with TalkBack enabled
- iPhone XR with TalkBack enabled 
- Any user

## Steps to Reproduce
For 1: 
1. Enable TalkBack on Android
2. Navigate to the What if my military service information doesn't look right page
3. Select the body text

For 2:
1. Enable TalkBack on Android
2. Navigate to the what if my military service information doesn't look right page
3. Select the entire page

## Desired Behavior
For 1:
- When the body text is selected, VA.gov is read as "VA [dot] gov"

For 2:
- When the page is selected, the programmatic title should read as "What if my military service information doesn't look right page"

## Acceptance Criteria
- See desired behavior

## Bug Severity
- 4 - Low

## Linked Story
- Story #13897

## Screen Shot (s) and Additional Information
1. Showing the affected text 🔽
![Screen Reader - Incorrect Service Info Text.png](https://images.zenhubusercontent.com/5f7bb216620a259ab7e50d60/bfe1577d-0c08-41a8-b4c1-912ce449bc0e)

2. Showing the affected area 🔽
![Screen Reader - Incorrect Service Info.png](https://images.zenhubusercontent.com/5f7bb216620a259ab7e50d60/aadd48e3-b580-4053-841e-397a2ceb2b71)

**Comment by digitalken** [comment]
PASS 2/4/2021 - Everything is being read correctly on Android and iOS.  Closing.
